dataflows-composed/mask-user-info/packages/mask-ssn/README.MD

```diff
@@ -21,7 +21,7 @@ The generator creates all necessary glue logic for a reusable package.
 
 ### Add Custom Code
 
-Let's update the content of `rust/mask-ssn/src/lib.rs` file with following code:
+Let's update the content of `rust/mask-ssn-pkg/src/lib.rs` file with following code:
```

Comment:
That should be in the function_name.rs instead of lib.rs

Reply:
updated.